fit.py

```diff
@@ -0,0 +1,296 @@
+
+# Mongo Database for FITNESS POT
+myclient = pymongo.MongoClient(os.environ.get("MONGO_CLUSTER"))
+mydb = myclient["fitness_pot"]
+dish_col = mydb["dish"]
+users_col = mydb["users"]
+collection_names = mydb.list_collection_names()
+
+
+DEBUG_LEVEL = "DEBUG"
+
+# =========
+# HOME PAGE - Display home page featured recipes with
+# image and introductory text only
+# =========
+
+
+@app.route("/", methods=["GET", "POST"])
+def index():
+    print(mydb.dish.find_one({"category": "Choose Category"}))
+    # this will allow to display 6 recipes per page. function located in
+    # utilities.py file
+    pagination = 6
+    recipes = mydb.dish.find()
+
+    selected_category = "All Recipes"
+
+    if request.args.get("category"):
+        selected_category = (
+            request.args.get("category").capitalize() + " Recipes"
+        )
+
+    if request.args.get("category"):
+        recipes = mydb.dish.find({"category": request.args.get("category")})
+
+    recipes, page, next = paginate(
+        recipes, pagination, request.args.get("page")
+    )
+    return render_template(
+        "index.html",
+        page=page,
+        recipes=recipes,
+        next=next,
+        categories=get_all_categories_from_db(),
+        selected_category=selected_category,
+        should_show_background_image=True,
+    )
+
+
+# =================================
+# CREATEUSER MODAL- function will allow create new account
+# ==================================
+@app.route("/create_user", methods=["POST"])
+def createuser():
+    regex = "^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$"  # noqa: W605
+    if not re.search(regex, request.form.get("email")):
+        flash("invalid email")
+        return redirect(url_for("index"))
+
+    newuser = {
+        "username": request.form.get("username"),
+        "password": request.form.get("password"),
+        "email": request.form.get("email"),
+    }
+    user = mydb.users.find_one({"username": request.form["username"]})
+    if user:
+        flash("Username already exists")
+    else:
+
+        try:
+            users_col.insert_one(newuser)
+            flash(
+                "Congratulation "
+                + request.form.get("username")
+                + "! You have created account"
+            )
+        # try and except to notify user in case there was some troubleshooting
+        except Exception:
+            flash(
+                "Sorry, there was some problem, user "
+                + request.form.get("username")
+                + " was not added to database"
+            )
+
+    return redirect(url_for("index"))
+
+
+# =================================
+# LOGIN MODAL- function will allow login to created account
+# ==================================
+@app.route("/login", methods=["POST", "GET"])
+def login():
+    login_user = mydb.users.find_one({"username": request.form["username"]})
+    if login_user:
+        if request.form["password"] == login_user["password"]:
```

Comment:
here you'll be comparing the password the user entered to the hashed password in your database